engine/crates/telemetry/src/metrics/operation.rs

```diff
@@ -23,6 +24,12 @@ pub struct GraphqlOperationMetrics {
     query_preparation_latency: Histogram<u64>,
     batch_sizes: Histogram<u64>,
     request_body_sizes: Histogram<u64>,
+    graphql_errors: Counter<u64>,
+}
+
+#[derive(Clone)]
```

Comment:
You can just remove the `Clone`, you don't need it for the response sender stuff in engine-v2, and `Clone` isn't even used in this repo. Don't think it is in api either.